src/main/java/com/opentable/server/ServerLoggingConfiguration.java

```diff
@@ -38,7 +42,12 @@
     public void onApplicationEvent(ApplicationEvent event) {
         if (event instanceof ApplicationEnvironmentPreparedEvent) {
             ApplicationEnvironmentPreparedEvent castEvent = (ApplicationEnvironmentPreparedEvent) event;
-            String serviceName = castEvent.getSpringApplication().getMainApplicationClass().getPackage().getImplementationTitle();
+            // Prefer OT, then Dmitry, then Manifest
```

Comment:
I thought that they're both from Dimitry?

Reply:
Nope see above